### ISSUE TITLE
incomplete logs when image is missing

### ISSUE THREAD

**Issue by EmilienM** [kubernetes-sigs/cluster-api-provider-openstack#2106]
/kind bug

if a Glance image is missing, this is the error that we get:
```
"Reconciler error" err="no images were found with the given image filter: {<nil> 0xc00071b0c0}" controller="openstackmachine" controllerGroup="infrastructure.cluster.x-k8s.io" controllerKind="OpenStackMachine" OpenStackMachine="default/without-lb-12343-control-plane-dwjgm" namespace="default" name="without-lb-12343-control-plane-dwjgm" reconcileID="370f585d-c19a-4117-bffb-fccf40c000cb"
```

It'll be better to provide the full filter so the user can know what image is missing.

**Comment by mdbooth** [comment]
That's a bug on the error message for sure.

The UX issue, btw, would be if we don't also push this something the user can view it.

**Comment by tsuzu** [comment]
seems to be resolved in #2135
